src/server.ts

```diff
@@ -35,28 +35,45 @@ const listenMessage = `Listening on port: ${config.APP_PORT}. ENV is ${config.EN
  * 5. Get movie by ID
  * */
 app.post("/movies", async (request: Request, response: Response) => {
-  const name = request.body.name;
+  const { name, comment, personalScore } = request.body;
   const data = await OmdbService.getMovie(name);
 
-  if (data["imdbID"]) {
-    movieStore.add(data);
+  // movie not found
+  if (!data["imdbID"]) {
+    // movie not found in IMDB - let's add it to local DB
+    const movie = { imdbID: movieStore.nextId(), ...request.body };
```

Comment:
This is a very unsafe decision. I can add to request body any information, for example, `"imdbID": 2` and then your database will have two movies with the same id. You already did body destructuring, so, please, use id and add only known and expected fields.